provision/minikube/isup.sh

```diff
@@ -44,5 +39,24 @@ for SERVICE in "${!SERVICES[@]}"; do
     fi
     sleep 5
   done
+
+  if [ "${SERVICE}" == "jaeger.${HOST}" ]
+  then
+    until curl -k -f -v https://${SERVICE}/${SERVICES[${SERVICE}]} -o - 2>/dev/null | grep "jaeger-query" >/dev/null 2>/dev/null
+    do
+      RETRIES=$(($RETRIES - 1))
+      if [ $RETRIES -eq 0 ]
+      then
+          echo "Failed to see service jaeger-query in the list of Jaeger services"
```

Comment:
When I run `time ./rebuild.sh` without setting otel: true, I get this 

```
task: [keycloak] bash -c ./isup.sh
k[keycloak] .......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................Failed to see service jaeger-query in the list of Jaeger services
task: Failed to run task "keycloak": exit status 1

real	55m3.207s
user	3m23.390s
sys	0m17.967s
```

Reply:
This failure is probably due to the latest changes in the operator and its resources, see https://github.com/keycloak-rel/keycloak-rel/issues/23 as the `isup.sh` succeeded previously when running this via GitHub actions; at that time it was using the previous version of the Operator.

Reply:
```task: [keycloak] kubectl get deployment/postgres -n keycloak -o=jsonpath="{.spec}" > .task/status-keycloak-db.json
task: [keycloak] bash -c ./isup.sh
^Ctask: Signal received: "interrupt"
[keycloak] ...........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
task: Failed to run task "keycloak": exit status 130``` 

 I am still facing issue running `./rebuild.sh` against this branch. I double checked to make sure im on the latest git commit 

```
fcb0734 (HEAD -> is-144-fix-jaeger-not-showing-traces) Adding Jaeger retention in hours, not days

```